### PR TITLE
Add a PR template

### PR DESCRIPTION
**Pull request by kprosise**
Added a PR template, which serves as "suggestions"; they are mostly
points that could have exceptions, and exist to serve as an aid.
As this is the first go, there will surely be changes made.

No QA steps taken.

Signed-off-by: Katrina Prosise <katrina.prosise@foundries.io>

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+_Why merge this PR? What does it solve?_
+
+## Checklist
+
+_Optional. Add a 'x' to steps taken._
+_You can fill this out after opening the PR. "Did I..."_
+
+* [ ] Avoid changing any header associated with a link/reference.
+* [ ] Step through instructions (or ask someone to do so).
+* [ ] Remove words that do not effect the meaning.
```
Comment (angolini) [inline .github/pull_request_template.md:12]: I think it difficult to know what this line means. because each word used will affect the meaning (because the word is there).
I would reword as something such as: Review to avoid redundant words and over description

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* [ ] Match tone and style of page/section.
+* [ ] Run spelling and Grammar check, preferably with linter.
```
Comment (angolini) [inline .github/pull_request_template.md:14]: I would put the line 14 as the very first item

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* [ ] Run `make linkcheck`.
+* [ ] View HTML in a browser to check rendering.
+* [ ] Consider using [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
```
Comment (angolini) [inline .github/pull_request_template.md:17]: I'm ok with this for now. But I don't like "consider" in a check list, I would say something like: Review for using semantic newlines

**Review by angolini — APPROVED** [review]
Great template!

Do we want to mention something about _ready to merge_ or _draft_ or _waiting for x_?

File: .github/pull_request_template.md (hunk 3 above)
Comment (kprosise) [reply .github/pull_request_template.md:17]: > Great template!
> 
> Do we want to mention something about _ready to merge_ or _draft_ or _waiting for x_?

I was thinking that could be mentioned in the comments section of the template, but now I am thinking a "Readiness/Current State" question could be useful! Thanks for the feedback.

File: .github/pull_request_template.md (hunk 1 above)
Comment (kprosise) [inline .github/pull_request_template.md]:
```suggestion
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
```
@angolini what about this link to an article with examples?

Comment (kprosise) [inline .github/pull_request_template.md:12]:
```suggestion
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
```
@angolini what about this link to an article with examples?

File: .github/pull_request_template.md (hunk 3 above)
Comment (kprosise) [inline .github/pull_request_template.md:17]:
```suggestion
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
```

Comment (kprosise) [inline .github/pull_request_template.md:17]:
```suggestion
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+## Overview
+
+_Why merge this PR? What does it solve?_
+
+## Checklist
+
+_Optional. Add a 'x' to steps taken._
+_You can fill this out after opening the PR. "Did I..."_
+
+ * [ ] Run spelling and grammar check, preferably with linter.
```
Comment (kprosise) [inline .github/pull_request_template.md:10]:
```suggestion
* [ ] Run spelling and grammar check, preferably with linter.
```

**Comment by doanac** [comment]
Docs for 93b6c5a5757bd7c8451ff931ea2f3990eebcde31 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1456/docs/artifacts/html/index.html

**Comment by kprosise** [comment]
I am going to go ahead and merge, and will add a readiness section in a separate PR.
